Tests/src/TestHelpers.h

```diff
@@ -30,6 +30,8 @@
 
 using namespace std::chrono_literals;
 
+using namespace std::chrono_literals;
```

Comment:
Duplicate line?

Reply:
Looks like it found it's way in while trying to fix the merge conflicts

Reply:
Understood. Ok to remove?